usaspending_api/common/query_with_filters.py

```diff
@@ -316,7 +359,12 @@ def generate_elasticsearch_query(cls, filter_values: List[dict], query_type: _Qu
                     raise InvalidParameterException(INCOMPATIBLE_DISTRICT_LOCATION_PARAMETERS)
                 if location_value is not None:
                     location_value = location_value.upper()
-                    location_query.append(ES_Q("match", **{f"recipient_location_{location_key}": location_value}))
+                    if query_type == _QueryType.SUBAWARDS:
```

Comment:
Recipient Location updates look good. Place of Performance Location still needs to be updated though. I think the Sub Place of Performance County Code might be missing from the ES index (I might have missed it though looking for the wrong name), so it may need to be added for this to work.

Reply:
Addressed.